variables.tf

```diff
@@ -334,6 +334,24 @@ variable "gm_license_key" {
   description = "The GrayMeta Platform license key. Contact support@graymeta.com if you have not been provided a license key"
 }
 
+variable "gm_scheduled_max_items" {
+  type        = "string"
+  description = "Max number of items in a harvest request. Default 100"
+  default     = "100"
+}
+
+variable "gm_scheduled_min_bytes" {
+  type        = "string"
+  description = "Minimum number of bytes in a harvest request.  Default 10485760 (10MB)"
+  default     = "10485760"
+}
+
+variable "gm_scheduled_wait_duration" {
+  type        = "string"
+  description = "The amount of time to wait before scheduling a harvest job. Default: 20000000000 (20 seconds)"
+  default     = "20000000000"
```

Comment:
set the default to `20s`. It uses a time.Duration, so will parse that correctly.